pinc/misc.inc

```diff
@@ -1622,3 +1622,20 @@ if (!class_exists('\ValueError')) {
     {
     }
 }
+
+function render_markdown_as_html(string $message): string
```

Comment:
This isn't a "useful exception" but it's in that section -- can we move it up somewhere else in the file? Perhaps under `sanitize_html()`. And let's add a docstring above it too please.

Reply:
Done.